src/mscorlib/corefx/System/Globalization/JapaneseCalendar.Win32.cs

```diff
@@ -7,12 +7,51 @@ namespace System.Globalization
 {
     public partial class JapaneseCalendar : Calendar
     {
-        public static int GetJapaneseEraCount()
+        private static EraInfo[] GetJapaneseEras()
         {
-            return WinRTInterop.Callbacks.GetJapaneseEraCount();
+            int erasCount = WinRTInterop.Callbacks.GetJapaneseEraCount();
```

Comment:
This code is unused for desktop.  The sources here are kept in sync manually with the .NET Native implementation.